pkg/operation/seed/seed.go

```diff
@@ -1036,6 +942,10 @@ func runCreateSeedFlow(
 
 	var (
 		g = flow.NewGraph("Seed cluster creation")
+		_ = g.Add(flow.Task{
+			Name: "Deploying Istio",
+			Fn:   istio.Deploy,
```

Comment:
Doesn't this yield a `nil` pointer exception when die feature gate is disabled (`istio` is `nil` in this case)?

Reply:
Yes, Will put a check based on feature gate.